zero_bin/rpc/src/lib.rs

```diff
@@ -19,6 +19,8 @@ pub mod retry;
 
 use compat::Compat;
 
+const PREVIOUS_HASHES_SIZE: usize = 256;
```

Comment:
I find the use of `size` here slightly confusing as it's not clear (without looking at the rest of the code) if this means the size in terms of bytes of a given hash or the count of hashes.